src/main/java/medicaldoctor/controllers/DoctorSearchController.java

```diff
@@ -0,0 +1,94 @@
+package medicaldoctor.controllers;
```

Comment:
whats the difference between DoctorSearch- and DoctorLookup- Controllers

Reply:
DoctorLookUp is the button that is located on the NavBar. doctor search is the controller that launches when you choose a specific doctor inside the DoctorLookUp page.
Example: You go to doctor lookup and find the doctor you want inside the tableview.
You double click and it will show the selected doctor in a new screen. This screen has information and the doctors current patients.

Reply:
Ok, i think it would normally be called a DoctorView. but this is fine.